core/src/main/java/org/infinispan/protostream/descriptors/AnnotatedDescriptor.java

```diff
@@ -41,17 +39,6 @@ public interface AnnotatedDescriptor {
     */
    String getDocumentation();
 
-   /**
-    * Get the documentation annotations in the form of a map of abstract syntax trees of {@link AnnotationElement} nodes.
-    * Only the configured annotations are returned. The unconfigured ones are discarded immediately after parsing.
-    * <p>
-    * The parsing of annotations is performed lazily on first invocation.
-    *
-    * @return the map of annotations
-    * @throws AnnotationParserException if parsing of annotations fails
-    */
-   Map<String, AnnotationElement.Annotation> getAnnotations() throws AnnotationParserException;
```

Comment:
Maybe we can keep the method implemented as as default simpy delegating to `getProcessedAnnotation("")`